examples/c/ex_all.c

```diff
@@ -345,7 +345,8 @@ cursor_ops(WT_SESSION *session)
 	cursor->set_key(cursor, key);
 	if ((ret = cursor->remove(cursor)) != 0) {
 		fprintf(stderr,
-		    "cursor.remove: %s\n", wiredtiger_strerror(ret));
+		    "cursor.remove: %s\n",
+		    cursor->session->strerror(cursor->session, ret));
```

Comment:
Because it's inside a snippet of code used as a documentation example, so I wanted it to be self-contained.